### PR TITLE
App views, Navbar and Headers

### LINKED ISSUE #38: Implement App Views
At the start, the mobile-app will have these views:
- Historical View
- Home View
- Settings View

With the Navigation Bar component, the users would be able to switch between these views easily.

### LINKED ISSUE #39: Implement Navigation Bar
This issue goes in hand with #38 where it would help users navigate through the app easily.

### LINKED ISSUE #40: Customise header
The React Navigation has a customizable Header that can be changed to include images. This allows us to put the app logo onto the header.

### PR DESCRIPTION
**Pull request by amrllkmn**
The pull request will add:
- App views:
    - Data ( for the historical data to be displayed )
    - Home ( indoor air quality will be rendered here)
    - Settings ( for users to customise the application )
- Navbar ( navigating between the views )
- Header ( the app's logo will be rendered here )

The PR closes issues #38, #39, #40.

### CODE REVIEW

**Review by roryort — COMMENTED** [review]
Only thing I would say is that the disclaimer at the top of each file needs to be implemented properly. I didn't see the disclaimer in each file and where I did see it it mentioned that it was for the monitoring server. If you can add the disclaimer for each file and ensure that you mention that it's for the App then I'll be good to approve. Other than that everything is good. (Sorry I know I am nit-picking here)

File: mobile-app/src/views/Settings.js [hunk 1]
```diff
@@ -20,12 +20,17 @@ import { StatusBar } from "expo-status-bar";
...
 function SettingsStackScreen() {
     return (
-        <SettingsStack.Navigator screenOptions={screenHeader}>
+        <SettingsStack.Navigator
+            screenOptions={{
+                headerTitle: () => <Header />,
+                headerStyle: styles.headerStyle,
+            }}
+        >
             <SettingsStack.Screen name="Home" component={Settings} />
```
Comment (Sevenarth) [inline mobile-app/src/views/Settings.js]: Settings not Home

**Review by Sevenarth — CHANGES_REQUESTED** [review]
Perhaps you could add a brief documentation for _nearly every_ function (?)

Globalise the stylesheets into a single file, so that there are not duplicates.

File: mobile-app/src/views/Settings.js (hunk 1 above)
Comment (amrllkmn) [reply mobile-app/src/views/Settings.js]: I'm resolving this conversation and re-requesting a review.

**Comment by amrllkmn** [comment]
Made the requested changes.

**Review by Sevenarth — APPROVED** [review]

**Review by roryort — APPROVED** [review]

**Pull request merged**